config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+root to: 'pets#index'
```

Comment:
Please add proper indentation :blush:
```suggestion
  root to: 'pets#index'
```